src/veh_type.cpp

```diff
@@ -412,6 +505,18 @@ std::string vpart_info::name() const
     return name_;
 }
 
+int vpart_info::install_time( const Character &ch ) const {
+    ///\EFFECT_MECHANICS reduces time consumed installing vehicle parts
+    int lvl = std::min( ch.get_skill_level( skill_mechanics ).level(), MAX_SKILL );
+    return install_moves * ( 1.0 - ( lvl / 2.0 ) / MAX_SKILL );
+}
+
+int vpart_info::removal_time( const Character &ch ) const {
+    ///\EFFECT_MECHANICS reduces time consumed removing vehicle parts
+    int lvl = std::min( ch.get_skill_level( skill_mechanics ).level(), MAX_SKILL );
+    return removal_moves * ( 1.0 - ( lvl / 2.0 ) / MAX_SKILL );
```

Comment:
Currently stub functions. No guarantee their behavior will remain equivalent. Adding a third function increases complexity for little simplification (three lines per function of which one is a doxygen comment).